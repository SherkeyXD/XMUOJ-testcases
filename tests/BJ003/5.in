3
2
86 
92 77 
35
74 
57 47 
53 75 98 
49 69 17 67 
29 79 50 28 73 
70 26 62 0 61 70 
35 40 61 10 79 25 56 
67 26 35 61 9 34 52 62 
4 76 90 88 22 49 48 17 26 
15 41 91 43 44 48 10 8 16 69 
69 96 25 66 14 55 31 41 80 17 0 
7 34 21 18 50 3 62 60 69 98 66 53 
10 46 92 3 87 31 25 78 86 9 42 38 76 
45 11 29 31 41 34 35 94 83 45 90 22 84 66 
95 86 24 88 24 16 58 59 8 74 75 86 11 22 22 
1 6 36 96 48 67 37 82 35 30 55 54 30 18 6 99 
83 47 33 56 67 88 81 61 30 51 88 45 72 18 27 51 57 
74 94 63 38 54 1 39 34 95 51 99 56 1 43 14 13 79 28 
47 95 22 40 55 21 75 68 57 75 64 81 10 86 82 52 50 54 64 
18 3 74 51 55 83 51 42 6 4 76 25 66 74 81 97 17 74 51 66 
85 26 16 36 94 78 38 48 77 3 83 98 9 90 9 59 28 2 11 36 68 
15 25 84 8 48 14 1 63 2 14 38 63 89 56 67 57 37 4 88 40 8 27 
24 52 70 43 65 88 57 71 84 29 97 71 11 28 70 95 16 56 38 96 32 46 99 
94 12 84 42 19 67 84 90 70 22 18 25 14 47 18 47 54 99 63 71 52 17 13 55 
93 35 84 59 6 34 23 14 2 84 6 52 25 60 65 14 43 31 5 8 86 3 40 63 64 
83 69 52 30 56 19 58 78 37 23 53 81 80 52 32 25 75 55 7 24 26 11 54 69 95 67 
42 77 57 29 48 40 38 47 95 36 53 78 40 58 49 4 65 58 35 85 78 50 41 49 78 79 75 
22 37 34 48 21 38 40 42 61 75 51 84 18 11 37 56 0 41 63 86 17 37 94 14 51 78 4 55 
9 1 76 12 86 51 19 32 10 61 75 7 47 93 84 88 32 6 69 33 67 42 23 8 98 33 72 94 71 
63 33 8 47 68 48 56 84 68 15 13 44 25 31 72 10 33 57 63 93 41 47 83 59 99 93 12 80 1 21 
2 25 4 70 88 49 84 55 51 74 61 61 58 70 89 47 35 61 55 44 38 1 65 44 90 60 68 55 54 70 7 
65 42 71 34 2 60 96 14 93 79 22 40 6 65 26 90 17 85 46 28 7 38 63 77 30 94 2 77 19 14 86 28 
40 83 28 38 11 34 73 98 10 63 66 39 48 21 39 25 7 68 61 22 78 39 10 21 26 63 45 37 39 93 20 79 91 
73 29 38 86 12 63 75 48 8 50 77 65 38 95 88 50 41 73 62 98 80 78 63 33 15 13 86 17 27 57 57 19 80 23 
92 29 89 96 5 36 45 8 42 91 48 98 61 90 77 50 91 35 76 2 58 74 83 19 64 50 73 54 24 3 98 8 16 24 46 
50
56 
65 13 
40 39 7 
65 12 5 45 
59 80 3 99 90 
25 8 56 66 68 31 
75 52 83 89 85 29 88 
44 21 84 67 33 53 33 70 
72 55 39 42 59 20 77 57 54 
73 9 55 86 50 58 67 0 35 53 
94 21 6 24 29 14 65 78 96 4 24 
80 93 49 15 14 87 2 91 22 19 67 8 
81 1 41 91 26 26 84 28 87 1 31 23 96 
92 54 90 80 54 49 74 1 34 45 45 39 77 11 
40 1 74 88 40 60 62 29 61 5 90 41 17 72 21 
80 23 66 41 42 6 87 78 6 65 73 90 85 82 0 43 
49 59 5 42 36 94 79 47 72 20 69 56 99 89 78 28 98 
80 16 17 83 32 73 70 51 67 63 21 48 95 86 15 11 30 76 
76 63 17 37 68 33 8 81 72 61 20 48 85 96 3 44 36 48 11 
95 12 90 43 44 57 77 43 14 13 44 42 51 9 14 13 12 45 76 20 
59 88 71 0 55 36 34 36 96 53 40 79 25 40 65 50 94 33 93 26 78 
25 78 40 19 27 21 1 92 94 63 48 44 84 67 36 85 54 13 85 20 77 73 
71 20 97 25 54 56 58 87 3 77 10 8 44 88 2 57 34 61 3 69 25 45 0 
68 93 13 29 93 4 6 52 51 78 63 9 62 31 45 15 79 31 96 16 78 69 82 77 
49 77 0 19 14 24 9 84 5 13 60 20 41 82 68 19 99 63 42 3 25 68 20 59 41 
21 51 22 22 4 79 76 69 65 59 5 11 71 3 91 26 84 91 75 6 88 92 37 98 94 87 
37 32 88 54 58 27 25 44 42 70 84 5 6 89 38 46 13 55 33 39 73 41 52 86 35 58 83 
10 30 22 15 75 91 67 74 75 91 5 83 42 14 78 6 7 53 46 42 72 66 85 69 33 50 80 89 
44 19 59 64 52 36 63 23 36 3 39 8 90 68 58 58 8 44 28 69 99 66 95 44 35 75 58 33 97 
54 4 66 58 19 53 87 57 7 1 44 84 29 15 8 4 24 88 40 68 35 25 33 61 69 83 76 8 36 63 
10 33 58 90 33 29 6 1 41 55 57 93 93 97 39 0 1 42 13 91 59 37 31 77 24 74 8 54 74 1 14 
78 62 11 10 58 87 47 72 58 56 53 87 55 74 20 51 5 72 33 45 92 53 87 76 14 44 73 42 79 7 39 78 
41 80 39 44 2 87 50 90 37 85 87 35 92 67 1 61 47 78 27 30 24 98 82 43 24 74 29 98 47 3 93 65 62 
73 47 38 98 46 78 67 73 51 53 25 51 0 95 8 50 40 1 41 84 22 19 38 74 8 57 31 10 35 22 58 24 90 51 
80 77 32 13 41 19 61 69 82 4 85 56 69 91 81 11 65 17 59 48 18 20 96 88 21 24 43 78 63 93 93 56 37 64 9 
87 17 85 64 57 12 12 16 65 97 67 56 70 9 5 50 39 12 96 0 77 16 98 35 67 2 38 80 25 89 57 97 54 48 47 98 
18 0 16 51 22 66 69 77 35 57 96 17 26 29 1 4 20 66 30 85 92 84 50 39 8 46 48 50 98 48 66 90 34 74 6 45 39 
39 49 58 28 98 0 94 33 94 4 25 49 44 29 87 17 20 35 59 46 79 24 64 22 94 94 3 24 91 68 84 91 87 49 4 86 72 5 
47 91 79 64 70 15 26 94 97 73 72 55 18 63 27 80 11 84 68 11 43 65 82 54 49 29 49 60 6 77 19 50 82 26 94 28 2 59 54 
58 79 93 35 91 15 17 4 6 10 96 92 51 98 11 60 28 91 23 41 64 74 38 66 73 28 99 2 48 49 11 30 54 46 83 92 21 44 96 18 
2 16 10 82 61 31 70 72 45 66 42 1 81 44 59 5 23 95 26 52 65 93 8 74 69 29 6 30 73 6 30 59 90 63 40 43 22 51 89 54 23 
72 67 42 33 97 82 48 45 71 70 33 83 20 54 25 92 45 28 4 72 79 54 77 63 58 86 95 43 74 42 44 0 11 55 86 18 62 91 94 7 56 90 
64 23 86 60 77 88 9 70 52 27 60 91 1 62 80 55 94 72 76 61 54 26 74 89 52 98 57 32 68 54 49 89 2 33 93 7 64 28 88 29 36 27 31 
70 27 58 61 65 21 1 41 17 79 46 51 82 7 59 70 12 69 36 55 24 36 40 58 91 60 93 9 56 18 39 62 27 76 63 68 68 13 49 72 77 81 6 16 
99 59 61 63 15 54 91 80 33 64 88 24 10 51 75 28 42 55 85 90 55 24 72 45 71 21 8 49 37 64 51 99 62 62 6 47 91 64 22 42 73 8 34 61 61 
68 6 66 73 20 94 10 81 55 69 47 47 88 96 91 51 98 12 16 41 27 94 26 77 87 25 91 32 89 54 77 40 18 41 10 47 81 93 91 52 61 32 85 28 53 30 
88 52 30 90 71 8 56 1 73 4 26 10 69 58 57 74 36 35 86 59 89 86 39 73 76 98 9 50 2 14 1 83 73 99 11 67 2 96 56 30 21 18 87 54 39 40 43 
50 72 51 97 49 11 14 4 4 18 99 15 51 54 0 60 10 65 36 99 79 88 70 28 81 81 98 27 6 26 9 58 39 68 6 83 84 60 96 94 69 41 8 99 94 90 23 12 
76 91 70 40 78 89 27 61 96 87 58 63 48 97 83 41 9 32 79 37 56 5 99 42 89 85 7 64 74 39 79 42 67 38 81 89 41 86 46 18 37 57 76 33 42 4 17 19 52 
60 20 13 43 25 79 44 14 99 6 9 10 74 78 21 14 3 50 63 20 68 49 26 57 68 82 93 30 23 98 68 98 9 5 1 39 79 59 13 57 48 51 20 7 76 51 2 10 9 38 
