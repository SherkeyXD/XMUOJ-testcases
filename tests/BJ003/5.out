178
2429
3728
